### DISCUSSION TITLE
Reposirory context of local folder config when running in docker

### DISCUSSION THREAD

**Discussion by mabhijithn** [TabbyML/tabby#1482] [General]
When running tabby using docker, how do I get repository context working? 
I added the local folder path in `~/.tabby/config.toml` . I entered the `bash` of docker container and ran the following command:
`tabby scheduler --now`. I get the following error: `Direcotory XXX does not exist`.
I understand the docker container cannot see my local directory. How can I do that?
It would be great if the [Configuration](https://tabby.tabbyml.com/docs/configuration) is updated for cases where `docker` is used to run Tabby.

**Comment by wsxiaoys** [comment]
Hey - you might find this tutorial useful:

https://docs.docker.com/guides/walkthroughs/access-local-folder/

**Comment by abhitoqua** [comment]
Perfect! Thanks!
